src/state.rs

```diff
@@ -96,10 +100,10 @@ impl AppState {
         event: WebhookNotificationEvent,
         account: &str,
     ) -> Result<(), NotifyServerError> {
-        if !is_valid_account(account) {
-            info!("Didn't register account - invalid account: {account} for project {project_id}");
-            return Err(NotifyServerError::InvalidAccount);
-        }
+        // if !is_valid_account(account) {
```

Comment:
What's the reason this commented out?

Reply:
Unrelated, I was getting IDE errors due to switching between branches on the `lazy_static` usage so I decided to just remove the dependency. It was only used by `is_valid_account()` and that's only used here but the webhook stuff is not in-use for now so decided to just delete/comment it out. Will replace with the `AccountId` type in the future when/if we re-introduce webhooks.